src/core_modules/capture-core/components/Pages/Enrollment/EnrollmentPageDefault/EnrollmentPageDefault.component.js

```diff
@@ -106,8 +107,10 @@ export const EnrollmentPageDefaultPlain = ({
                     teiId={teiId}
                     enrollmentId={enrollmentId}
                     programId={program.id}
+                    mayEditDate
```

Comment:
Can we go with `readOnlyMode` even though the mode is not complete in this PR? I understand your thinking, but I think it will be easier for someone to finish up this later if we introduce it here.

And `readOnlyMode` should be optional (defaults to false). Meaning you can omit it in this file.